.gitignore

```
@@ -8,6 +8,7 @@
 *id_rsa.pub
 id_rsa*.pub
 *.swp
+*.dtmp
```

what file type is that?
```suggestion
*.dtmp
```

---

that are temp files for the diagrams